### PR TITLE
pd_real_cluster: remove cd to specific dir

### PR DESCRIPTION
**Pull request by HuSharp**
after https://github.com/tikv/pd/pull/7567 merged we can remove `cd` operator

### CODE REVIEW

**Comment by HuSharp** [comment]
cc @JmPotato

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 1]
```diff
@@ -73,7 +73,6 @@ pipeline {
             steps {
                 dir('pd') {
                     sh label: "PD Real Cluster Check", script: """
-                        cd tests/integrations/realtiup
                         make check
```
Comment (JmPotato) [inline pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:76]: Should be `make test-real-cluster`.

Comment (HuSharp) [reply pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy]: 0 0 fixed

**Review by JmPotato — APPROVED** [review]

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**